encfs/encfsctl.cpp

```diff
@@ -59,6 +59,7 @@ static int showInfo(int argc, char **argv);
 static int showVersion(int argc, char **argv);
 static int chpasswd(int argc, char **argv);
 static int chpasswdAutomaticly(int argc, char **argv);
+static int ckpasswdAutomaticly(int argc, char **argv);
```

Comment:
No, these are 2 differences methods, `chpasswdAutomaticly`to change, `ckpasswdAutomaticly`to only check 👍